src/app/static/src/app/apiService.ts

```diff
@@ -74,19 +74,20 @@ export class APIService<S extends string, E extends string> implements API<S, E>
     };
 
     withSuccess = (type: S) => {
-        this._onSuccess = (success: Response) => {
-            this._commit({type: type, payload: freezer.deepFreeze(success.data)});
+        this._onSuccess = (data: any) => {
+            this._commit({type: type, payload: freezer.deepFreeze(data)});
         };
         return this;
     };
 
     private _handleAxiosResponse(promise: Promise<AxiosResponse>) {
         return promise.then((response: AxiosResponse) => {
             const success = response && response.data;
+            const data = success.data;
             if (this._onSuccess) {
-                this._onSuccess(success);
+                this._onSuccess(data);
             }
-            return success;
+            return data;
```

Comment:
Realised it made more sense to just return the data from the response here, rather than the whole wrapped success response. This wasn't actually being used anywhere yet, [this](https://github.com/mrc-ide/hint/pull/139/files#diff-cbdcdbb269d6697eb9210ac538053e44R39) is the first instance of it.